server/blindai_sgx/src/client_communication.rs

```diff
@@ -145,20 +147,30 @@ impl Exchange for Exchanger {
             input_facts.push(input_fact.clone());
         }
 
+
+        // Optimize, save and seal
+        let model_id = Uuid::new_v4();
+        let mut models_path = std::env::current_dir()?;
```

Comment:
I would like this to be configurable: probably in the config.toml file or something.